way_to_home/tests/unittests/way/testmodel.py

```diff
@@ -0,0 +1,113 @@
+"""This module provides tests for Way model"""
+
+import datetime
+
+from django.test import TestCase
+from way.models import Way
+from custom_user.models import CustomUser
+from place.models import Place
+from route.models import Route
+
+
+
+class WayModelTestCase(TestCase):
+	"""TestCase for providing Way model testing"""
+
+	def setUp(self):
+		"""Method that provides preparation before testing Way model's features."""
+		user = CustomUser.objects.create(id=100, email='mail@gmail.com', password='Password1234', is_active=True)
+		start_place = Place.objects.create(id=100, longitude=111.123456, latitude=222.123456)
+		end_place = Place.objects.create(id=200, longitude=222.123456, latitude=111.123456)
+
```

Comment:
longtitude must be between -180 and 180, and latitude must be betweent -90 and 90, else you might run into issues with place_data_validator